app/propertyLoaders/staticConfigServerPropertyLoader.ts

```diff
@@ -0,0 +1,25 @@
+import { PropertyLoader } from '../propertyLoaders/propertyLoader';
+import { PropertySource } from '../propertySources/propertySource';
+
+interface configServerPropertySource {
+  Name: string;
+  Source: any;
```

Comment:
The source property should be a `{[key: string]: string | number | bool}` type right?